packages/pulp-rpm/pulp-rpm.spec

```diff
@@ -269,7 +278,25 @@ A collection of yum plugins supplementing Pulp consumer operations.
 %{_usr}/lib/yum-plugins/pulp-profile-update.py*
 %doc LICENSE COPYRIGHT
 
+# ---- Pulp Manifest -------------------------------------------------------------
+%if %{pulp_server}
```

Comment:
Why the restriction here?

Reply:
pulp-rpm gets built for el5/el6 as well.  Since pulp_server is set at the top of the .spec based on those distros, I'm using it here as well to ensure it gets built only for the neccessary platforms.